draft-dekater-scion-dataplane.md

```diff
@@ -1353,6 +1353,18 @@ This section describes the steps that a SCION egress border router MUST perform
 **Note:** For more information on the path meta header, see [](#PathMetaHdr).
 
 
+#### Effects of Clock Inaccuracy
+
+Each router along a packet's path verifies the validity of the current hop field by comparing the current time with the hop's expiration time.
+
+This expiration time is calculated as described in [](#hopfld) on the basis of the segment's timestamp. That timestamp is assigned by the host that originates the segment. A fast clock at origination or a slow clock at a router will yield a lengthened time-to-live; possibly an origination time in the future. A slow clock at origination or a fast clock at a router will yield a shortened time to live; possibly an expiration time in the past.
+
+By default, segments are propagated once a minute. A segment is registered by the last AS of that segment, therefore up to N minutes after origination, where N is the length of the segment. As a result, a segment must have a time-to-live of at least N minutes to be of any use. N being the length of the segment. On the other hand, a very recent segment, which clocks offset could make appear from the future, ages by 1 minute per hop before being used, thereby reducing the impact of clock drift in that respect.
+
+The unit of a segments time-to-live is 5 minutes and 37 seconds (and 500 ms), or the equivalent of 5 hops. Given the above constraints, it is unreasonable to create a segment with a short time-to-live, while each additional time-to-live unit adds more than 5 minutes. As a result, a clock drift of up to 1 minute can be safely neglected.
+
+Each administrator of a SCION router or core control service is responsible for maintaining sufficient clock accuracy. No particular method is assumed by this specification.
+
```

Comment:
I'd also shrink and reword this section, after we finish discussion in the control plane PR, and leave here only aspects that relate to the DP, like: 
- there must be lose time sync between routers and CPs (within an AS?) because routers must verify the current HF 

And remove parts that relate to CP

Reply:
done.